robotiq_force_torque_sensor/CMakeLists.txt

```diff
@@ -10,12 +10,14 @@ find_package(catkin REQUIRED COMPONENTS
 
 ## Generate messages in the 'msg' folder
 add_message_files(
-   FILES
-   ft_sensor.msg
+  DIRECTORY msg
+  FILES
+  ft_sensor.msg
```

Comment:
Specifying the directory here should not be necessary (as `msg` is the default), but it doesn't hurt either. Personally I like to be as explicit as possible.